packages/devtools_app/test/flutter/story_of_layout/flex_test.dart

```diff
@@ -14,7 +14,7 @@ import 'package:flutter_test/flutter_test.dart';
 import '../wrappers.dart';
 
 void main() {
-  const windowSize = Size(1500, 1500);
+  const windowSize = Size(1750, 1750);
```

Comment:
this is intended because I need more height to show the constraints text in the "height indicator arrow". For some reason the test is overflowing even though it should not.